lib/mongo/database.rb

```diff
@@ -155,11 +158,11 @@ def collections
     def command(operation, opts = {})
       preference = ServerSelector.get(opts[:read] || ServerSelector::PRIMARY)
       server = preference.select_server(cluster)
-      Operation::Commands::Command.new({
+      Operation::Commands::Command.new(
```

Comment:
Thanks, I've written a comment in the google doc next to the text saying it reports it from CRUD commands.